draft-ietf-quic-transport.md

```diff
@@ -4028,7 +4028,7 @@ smallest allowed maximum packet size of 1200 bytes.
 
 UDP datagrams MUST NOT be fragmented at the IP layer.  In IPv4
 ({{!IPv4=RFC0791}}), the DF bit MUST be set to prevent fragmentation on the
-path.
+path, if possible.
```

Comment:
gh doesn't let me suggest a change on unmodified lines: "... the DF bit MUST be set if possible, to prevent fragmentation on the path."